### PR TITLE
move p2p test to their own test suite

### PR DESCRIPTION
**Pull request by beckmani**

### CODE REVIEW

**Comment by y0sher** [comment]
I don't see the p2p tests here I think you forgot to add them

File: tests/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-__all__ = ['test_bs', 'misc', 'fixtures']
+__all__ = ['test_bs', 'test_p2p', 'misc', 'fixtures']
```
Comment (y0sher) [inline tests/__init__.py]: why do we need this for p2p but not for hare, sync, etc.. ?

File: Makefile [hunk 2]
```diff
@@ -157,6 +157,11 @@ ifndef ES_PASSWD
 	$(error ES_PASSWD is not set)
 endif
 
+	docker run -e ES_PASSWD="$(ES_PASSWD)" \
+             -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json \
+             -e CLIENT_DOCKER_IMAGE="spacemeshos/$(DOCKER_IMAGE_REPO):$(BRANCH)" \
+             -it go-spacemesh-python pytest -s test_p2p.py$(DELIM)$(TEST) --tc-file=config.yaml --tc-format=yaml
```
Comment (y0sher) [inline Makefile]: why do it in a separate file and not a separate folder like hare or sync?

File: tests/__init__.py (hunk 1 above)
Comment (beckmani) [reply tests/__init__.py]: Basically we don't need it at this stage. We might need it in the future.
Removing it for now...

File: Makefile (hunk 2 above)
Comment (beckmani) [reply Makefile]: It is just easier. I'll move it to new directory

**Comment by beckmani** [comment]
> I don't see the p2p tests here I think you forgot to add them

You are right. forgot it

File: Makefile [hunk 3]
```diff
@@ -160,7 +160,12 @@ endif
 	docker run -e ES_PASSWD="$(ES_PASSWD)" \
              -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json \
              -e CLIENT_DOCKER_IMAGE="spacemeshos/$(DOCKER_IMAGE_REPO):$(BRANCH)" \
-             -it go-spacemesh-python pytest -s test_bs.py$(DELIM)$(TEST) --tc-file=config.yaml --tc-format=yaml
+             -it go-spacemesh-python pytest -s p2p/test_p2p.py$(DELIM)$(TEST) --tc-file=config.yaml --tc-format=yaml
```
Comment (y0sher) [inline Makefile]: again, I don't understand why this is run differently (` p2p/test_p2p.py$(DELIM)$(TEST)` ) then the below lines of hare and python

Comment (y0sher) [reply Makefile]: @beckmani ?

**Review by y0sher — CHANGES_REQUESTED** [review]
please address my last comment and I'll approve.

**Review by y0sher — APPROVED** [review]

**Pull request merged**